src/components/Footer.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import styled from "styled-components";
+
+const FooterWrapper = styled.div`
+  position: fixed;
```

Comment:
I don't think the footer needs to be fixed (e.g always visible), it should be placed at the very bottom but if the content grows it needs to be pushed down (like any other footer). 
Also, it causes an overlap with the main content:
<img width="490" alt="Screen Shot 2019-07-30 at 10 39 51" src="https://user-images.githubusercontent.com/14864439/62114773-9a5c2480-b2b7-11e9-87c2-966adfcb2f98.png">